tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -2204,6 +2206,44 @@ def get_valid_tactics(
 
         return valid_strategies
 
+    def _register_native_tactics(self, input: torch.Tensor) -> None:
```

Ruff (0.15.21)

[error] Function argument `input` is shadowing a Python builtin

(A002)